jupyter_dashboards/nbextension/notebook/dashboard-view/dashboard-view.js

```diff
@@ -112,6 +120,8 @@ define([
                     if ($helpArea) {
                         // remove if it exists since layout-specific help text will be inserted
                         $helpArea.remove();
```

Comment:
I think `remove` only cleans up event handlers on the target element, the help area in this case. Any handlers registered on descendant elements (the checkbox and more info link in this case) leak I believe.

>The only way that a $.cache entry gets deleted is when you call remove() on the element that put it there!` 

from http://makandracards.com/makandra/31325-how-to-create-memory-leaks-in-jquery

I believe this is why jQuery support for bubbling exists. You register handlers on stable elements so you don't have to worry about registering and unregistering the listeners as content comes and goes.